sos/report/plugins/kubernetes.py

```diff
@@ -228,6 +229,9 @@ def setup(self):
         for svc in self.services:
             self.add_journal(units=svc)
 
+        if self.is_installed('microk8s'):
+            self.kube_cmd = 'microk8s kubectl'
```

Comment:
Just testing, and realised the `_kconf` files never got used, and I think, we out to have this if statement before the for loop above. This will ensure that the following command would run

```
microk8s kubectl --kubeconfig=/var/snap/microk8s/current/credentials/client.config
```

Would that make sense? or does `microk8s kubectl` already take care of this?

Reply:
Hi @arif-ali, `microk8s kubectl` already calls kubectl with admin-level access. I think we do not actually need the client.config at all.

```
$ sudo snap install --classic --channel=latest/edge microk8s
microk8s (edge) v1.29.1 from Canonical✓ installed
$ sudo microk8s kubectl get pods -A
NAMESPACE     NAME                                     READY   STATUS     RESTARTS   AGE
kube-system   calico-node-tmkg7                        0/1     Init:0/2   0          1s
kube-system   calico-kube-controllers-77bd7c5b-98x9p   0/1     Pending    0          1s
kube-system   coredns-864597b5fd-4pq88                 0/1     Pending    0          1s
```